pkg/scraper/scraper_test.go

```diff
@@ -259,17 +265,24 @@ func (c *fakeKubeletClient) GetMetrics(ctx context.Context, node *corev1.Node) (
 	return metrics, nil
 }
 
+type cacheStore struct {
```

Comment:
I don't understand purpose of this struct. 
In my opinion there is no reason to test the scraper here. It does no logic related to label selector, it only takes it as argument and later passes down. It's too simple thing to test. 
I would recommend rather implement a e2e test.

Reply:
Yeah， maybe I can add some e2e test, Let me update it.